src/components/pages/test-pages/controls-page/ControlsTestPage.tsx

```diff
@@ -59,25 +59,24 @@ const ControlsTestPage = () => {
             />
           </Container>
           <br />
-          <RadioGroup sx={styles.wrapper} onChange={handleChange}>
-            <Radio
-              name="r1"
-              value="1"
-              label="text them 1"
-              textType="body2Medium"
-            />
-            <Radio name="r2" value="2" label="text them 2" />
-            <Radio name="r3" value="2" label="text them 2" disabled={true} />
-            <Radio
-              textType="body2Medium"
-              name="r4"
-              value="3"
-              label="text them 3"
-            />
-            <Radio name="r5" value="4" label="text them 4" />
-            <Radio name="r6" value="5" label="text them 5" />
-            <Radio name="r7" value="6" label="text them 6" />
-          </RadioGroup>
+          <RadioGroup
+            onChange={handleChange}
+            options={[
+              { value: '1', label: 'так' },
+              { value: '0', label: 'ні' },
+            ]}
+            textType="body1"
+            name="r1"
+          />
+          {/* <RadioGroup name="r1" sx={styles.wrapper} onChange={handleChange}>
+            <Radio value="1" label="text them 1" textType="body2Medium" />
+            <Radio value="2" label="text them 2" />
+            <Radio value="2" label="text them 2" disabled={true} />
+            <Radio textType="body2Medium" value="3" label="text them 3" />
+            <Radio value="4" label="text them 4" />
+            <Radio value="5" label="text them 5" />
+            <Radio value="6" label="text them 6" />
+          </RadioGroup> */}
```

Comment:
If it is unnecessary, delete it, if it is necessary, rewrite the logic or add a method.